etl/extractor/platform_impl.go

```diff
@@ -116,6 +117,11 @@ func (m *PlatformExtractor) getJetDrops(ctx context.Context, request *exporter.G
 						log.Warnf("no data in the pulse. waiting for pulse will be changed. sleep %v", pulseDelta)
 						time.Sleep(time.Second * pulseDelta)
 					}
+					// if we have received all data for topSyncPulse and the pulse didn't change yet, we need to continue
+					// when it happened it means that we need to request again without resetting the pulse number
+					if lastPulseNumber == receivedPulseNumber {
```

Comment:
Maybe do sleep here for some time, one second maybe? We call to platform very often if its just `continue`

Reply:
good idea. thank you